### PR TITLE
Feature/kotis/add player functionality

### PR DESCRIPTION
**Pull request by kotisvarand**
1.Add
2.Player
3.xyz

### CODE REVIEW

File: ViewControllers/ViewController.swift [hunk 1]
```diff
@@ -9,6 +9,9 @@ import UIKit
 
 class ViewController: UIViewController {
     
+    //Add player done
+    //Add player done
+
```
Comment (Ajinkya3090) [inline ViewControllers/ViewController.swift]: Remove extra line

File: ViewControllers/ViewController.swift [hunk 2]
```diff
@@ -9,6 +9,9 @@ import UIKit
...
     @IBOutlet weak var TitileSegView: UISegmentedControl!
```
Comment (Ajinkya3090) [inline ViewControllers/ViewController.swift:14]: Add mark statements

Comment (kotisvarand) [reply ViewControllers/ViewController.swift:14]: Resolved

File: ViewControllers/ViewController.swift (hunk 1 above)
Comment (kotisvarand) [reply ViewControllers/ViewController.swift]: Done

**Review by Ajinkya3090 — APPROVED** [review]
You can merge

**Pull request merged**